### PR TITLE
Active Job: Add note about ability to configure adapters on per job basis in CHANGELOG and release notes

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**
r? @jeremy

### CODE REVIEW

File: activejob/CHANGELOG.md [hunk 1]
```diff
@@ -80,6 +80,23 @@
...
 
+*   Add ability to configure the queue adapter on per job basis.
+
+    Now different jobs can have different queue adapters without conflicting with
+    each other.
+
+    Example:
+
+         class EmailJob < ActiveJob::Base
+            self.queue_adapter = :sidekiq
```
Comment (kaspth) [inline activejob/CHANGELOG.md]: One space too many 😁

File: activejob/CHANGELOG.md [hunk 2]
```diff
@@ -80,6 +80,23 @@
 
     *Jeroen van Baarsen*
 
+*   Add ability to configure the queue adapter on per job basis.
```
Comment (kaspth) [inline activejob/CHANGELOG.md]: `on a per job basis`

File: guides/source/5_0_release_notes.md [hunk 3]
```diff
@@ -655,11 +655,15 @@ Please refer to the [Changelog][active-job] for detailed changes.
 
 ### Notable changes
 
-*   `ActiveJob::Base.deserialize` delegates to the job class. this allows jobs
+*   `ActiveJob::Base.deserialize` delegates to the job class. This allows jobs
     to attach arbitrary metadata when they get serialized and read it back when
     they get performed.
     ([Pull Request](https://github.com/rails/rails/pull/18260))
 
+*   Add ability to configure the queue adapter on per job basis. This allows to have
```
Comment (kaspth) [inline guides/source/5_0_release_notes.md]: Needs an `on a` here as well. Don't think the second sentence adds anything to what the first one said. Except for the `affecting each other` it's just a rephrase — let's try integrating that part into the first sentence.

**Comment by prathamesh-sonpatki** [comment]
@kaspth :+1: Thanks, I updated.

**Comment by kaspth** [comment]
❤️💙💚

**Pull request merged**